docs/source/_topic_map.yml

```diff
@@ -14,8 +14,8 @@ Topics:
         File: installing
       - Name: Quickstart
         File: quickstart
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
+      - Name: Setting up the hypervisor drivers
```

Comment:
I disagree, as Minikube calls this https://github.com/kubernetes/minikube/blob/master/docs/drivers.md "Driver plugin installation". The reasoning of the setup is related to the fact that also virtual switch settings are considered (due to Hyper-V). If at best, the page can be named: "Setting up the docker-machne drivers", but this is specifically referring to Docker and Docker machine, while these as technology should be mostly hidden from the user. Especially going forward, I believe we aregoing to pack them into the binary (if possible).

Even Docker Machine calls is Machine plugins and driver plugins to be precise: https://github.com/docker/docker.github.io/blob/master/machine/AVAILABLE_DRIVER_PLUGINS.md.

While I partially agree, I also disagree as much.